web/components/Section/Header/Button/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { ComponentProps } from "react";
+import clsx from "clsx";
+
+type ButtonProps = ComponentProps<"div"> & {};
+
+export const Button = (props: ButtonProps) => {
+  const { className, ...otherProps } = props;
+  return (
+    <div
```

Comment:
@michaelfurutin Is it possible to use common/Button component here instead of simple div to make all buttons consistent and have common interface.

Reply:
Not a big problem, but when you are using element called `Button`, you expect it to be button by default